core/domain/skill_services_test.py

```diff
@@ -1143,36 +1234,37 @@ def test_migrate_rubrics_to_latest_schema(self):
             misconceptions=[],
             rubrics=[{
                 'difficulty': 'Easy',
-                'explanation': 'Easy explanation'
+                'explanations': ['Easy explanation']
             }, {
                 'difficulty': 'Medium',
-                'explanation': 'Medium explanation'
+                'explanations': ['Medium explanation']
             }, {
                 'difficulty': 'Hard',
-                'explanation': 'Hard explanation'
+                'explanations': ['Hard explanation', html]
             }],
             skill_contents=skill_contents.to_dict(),
             next_misconception_id=1,
             misconceptions_schema_version=1,
-            rubric_schema_version=1,
-            skill_contents_schema_version=1,
+            rubric_schema_version=2,
+            skill_contents_schema_version=2,
             all_questions_merged=False
         )
         commit_cmd_dicts = [commit_cmd.to_dict()]
         model.commit(
             'user_id_admin', 'skill model created', commit_cmd_dicts)
 
-        swap_skill_object = self.swap(skill_domain, 'Skill', MockSkillObject)
+        swap_skill_object = self.swap(skill_domain, 'Skill', skill_domain.Skill)
```

Comment:
is this an expected change?

Reply:
Yes we have removed the MockSkillObject

Reply:
Then remove the line completely? [Why mock to a correct value?]

Reply:
done